go/interpreter/lfvm/interpreter.go

```diff
@@ -604,15 +603,13 @@ func steps(c *context, oneStepOnly bool) {
 	}
 }
 
+// satisfiesStackRequirements checks if the current stack size satisfies the
```

Comment:
this sentence does not say much, does it? 
```suggestion
// satisfiesStackRequirements checks that the opCode will not make an out of bounds access with the current stack size. 
```